internal/cli/auth/login.go

```diff
@@ -306,6 +310,7 @@ func LoginBuilder() *cobra.Command {
 		Args: require.NoArgs,
 	}
 
+	cmd.Flags().BoolVar(&opts.isCloudManager, "cm", false, "Log in to Cloud Manager.")
```

Comment:
What does it mean having this flag as false? where are we logging in? it is either atlas or cm today.

perhaps we should indeed remove the flag and assume it is only for cm.

Reply:
That's what I originally did, but @gssbzn asked me to put it back.